app/controllers/scim_rails/application_controller.rb

```diff
@@ -59,7 +59,13 @@ def extract_from_inside_quotations(string)
     end
 
     def extract_path_params(operation)
-      operation.key?("path") ? process_path(operation) : nil
+      return nil unless operation.key?("path")
+      
+      if operation["path"].include?("urn:ietf:params:scim:schemas:extension:enterprise:2.0:User")
+        return process_extension_schema_path(operation) 
+      end
```

Comment:
If there are more edge cases like this, then we might wanna think about duck typing it.